src/actmonster.cpp

```diff
@@ -4939,3 +4943,89 @@ void handleMonsterSpecialAttack(Entity* my, Stat* myStats, Entity* target, doubl
 	return;
 }
 
+void getTargetsAroundEntity(Entity* my, Entity* originalTarget, double distToFind, real_t angleToSearch, int searchType, list_t** list)
```

Comment:
As a rule of thumb, anything taking a my argument probably belongs as a member of that class.